File: SecondYear/praptisharma28_PraptiSharma_2226CSE(AI&ML)1008_2/Week_1/IF-Else.py
from typing import *

def compare(a: int, b: int) -> str:
    comparison_result = {
        a == b: 'equal',
        a > b: 'greater',
        a < b: 'smaller'
    }
    return comparison_result[True]

print(compare(3, 8))   

File: SecondYear/LakshayJain458_LakshayJain_2226cs1199_2/Week_1/ifelse.c
#include <stdio.h>
int main()
{
    int a, b;
    scanf("%d%d", &a, &b);
    if (a > b)
    {
        printf("Greater");
    }
    else if (a < b)
    {
        printf("smaller");
    }
    else
    {
        printf("equal");
    }
    return 0;
}